fastlane/metadata/android/en-US/changelogs/16.txt

```diff
@@ -0,0 +1,15 @@
+🚀 Features and Enhancements
+- feat(ui): Add event name and time in dashboard
+- feat: Add 'About The App' section
+- feat: Add bio section in Edit Profile
+- feat: Option to delete profile picture
```

Comment:
Why? Do not commit this file.

Reply:
I am so sorry. I didn't commited the file intentionally. I am new to git and github. So, that is a mistake.

Reply:
@liveHarshit I have deleted the file. Sorry for the mistake sir!